Makefile

```diff
@@ -0,0 +1,104 @@
+
+$(XSHARP_DIR):
+	@echo "Cloning Cosmos/XSharp"
+	$(GIT) $(GITFLAGS) --branch=$(XSHARP_BRANCH) $(XSHARP_URL) $(THISDIR)/../XSharp
+
+$(COMMON_DIR):
+	@echo "Cloning Cosmos/Common"
+	$(GIT) $(GITFLAGS) --branch=$(COMMON_BRANCH) $(COMMON_URL) $(THISDIR)/../Common
+
+.PHONY: build
+build:
+	@echo "Building IL2CPU"
+	$(DOTNET) build $(IL2CPU_DIR) $(DOTNETFLAGS)
+	$(DOTNET) pack $(IL2CPU_DIR) $(DOTNETFLAGS)
+
+	@echo "Building Cosmos"
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Common $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Debug.Kernel $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Debug.Kernel.Plugs.Asm $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Core $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Core_Asm $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Core_Plugs $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.HAL2 $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.System2 $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.System2_Plugs $(DOTNETFLAGS)
+	$(DOTNET) pack $(THISDIR)/source/Cosmos.Build.Tasks $(DOTNETFLAGS)
+
+.PHONY: publish
+publish:
+	@echo "Publishing IL2CPU"
+	$(DOTNET) publish $(IL2CPU_DIR)/source/IL2CPU -r linux-x64 --self-contained $(DOTNETFLAGS)
+
+	@echo "Publishing Cosmos"
+	$(DOTNET) publish $(THISDIR)/source/Cosmos.Core_Plugs $(DOTNETFLAGS)
+	$(DOTNET) publish $(THISDIR)/source/Cosmos.Debug.Kernel.Plugs.Asm $(DOTNETFLAGS)
+	$(DOTNET) publish $(THISDIR)/source/Cosmos.HAL2 $(DOTNETFLAGS)
+	$(DOTNET) publish $(THISDIR)/source/Cosmos.System2_Plugs $(DOTNETFLAGS)
+
+.PHONY: install
+install:
+	@echo "Installing to" $(DESTDIR)
+	@mkdir -p $(DESTDIR)/Cosmos
+	@mkdir -p $(DESTDIR)/XSharp/DebugStub
+	@mkdir -p $(DESTDIR)/Build/ISO
+	@mkdir -p $(DESTDIR)/Build/IL2CPU
+	@mkdir -p $(DESTDIR)/Build/HyperV
+	@mkdir -p $(DESTDIR)/Build/VMware/Workstation
+	@mkdir -p $(DESTDIR)/Packages
+	@mkdir -p $(DESTDIR)/Kernel
+	@cp -r $(IL2CPU_DIR)/artifacts/Debug/nupkg/*.nupkg $(DESTDIR)/Packages/
+	@cp -r $(THISDIR)/artifacts/Debug/nupkg/*.nupkg $(DESTDIR)/Packages/
+	@cp -r $(IL2CPU_DIR)/source/Cosmos.Core.DebugStub/*.xs $(DESTDIR)/XSharp/DebugStub/
+
+	@cp -r $(THISDIR)/Artwork/XSharp/XSharp.ico $(DESTDIR)/XSharp/
+	@cp -r $(THISDIR)/Artwork/Cosmos.ico $(DESTDIR)/
+
+	@cp -r $(IL2CPU_DIR)/source/IL2CPU/bin/Debug/net5.0/linux-x64/publish/* $(DESTDIR)/Build/IL2CPU/
+	@cp -r $(THISDIR)/source/Cosmos.Core_Plugs/bin/Debug/net5.0/publish/*.dll $(DESTDIR)/Kernel/
+	@cp -r $(THISDIR)/source/Cosmos.System2_Plugs/bin/Debug/net5.0/publish/*.dll $(DESTDIR)/Kernel/
+	@cp -r $(THISDIR)/source/Cosmos.HAL2/bin/Debug/net5.0/publish/*.dll $(DESTDIR)/Kernel/
+	@cp -r $(THISDIR)/source/Cosmos.Debug.Kernel.Plugs.Asm/bin/Debug/netstandard2.0/publish/*.dll $(DESTDIR)/Kernel/
+
+	@cp -r $(THISDIR)/build/HyperV/*.vhdx $(DESTDIR)/Build/HyperV/
+	@cp -r $(THISDIR)/build/VMWare/Workstation/* $(DESTDIR)/Build/VMware/Workstation/
+	@cp -r $(THISDIR)/build/syslinux/* $(DESTDIR)/Build/ISO/
```

Comment:
Fix `build` to `Build`

Reply:
yasm is in build folder but the make file is confused so it uses build but it should use Build folder instead